src/components/Button.tsx

```diff
@@ -0,0 +1,34 @@
+import { FontType } from 'styled-components'
+import { BaseButton } from './styles/Button.styled'
+
+type Props = {
+  text: string
+  hasActive?: boolean
+  link?: string | undefined
```

Comment:
no need to specify undefined type
```suggestion
  link?: string
```

Reply:
Resolved in commit 8cb1f2e3fa2ff50e16fa4367e6681ec6e97021c9.